configs/platforms/osx-10.13-x86_64.rb

```diff
@@ -15,5 +15,5 @@
   plat.provision_with '/usr/local/bin/osx-deps pkg-config'
   plat.install_build_dependencies_with "/usr/local/bin/osx-deps "
   plat.vmpooler_template "osx-1012-x86_64"
-  plat.output_dir File.join("apple", "10.13", "PC1", "x86_64")
+  plat.output_dir File.join("apple", "10.13", "puppet5", "x86_64")
```

Comment:
If you want these to look like what we ship publicly, the repo comes between the platform and version now (for mac and eos):
http://downloads.puppetlabs.com/mac/puppet5/10.13/x86_64/